WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -412,6 +417,59 @@ private extension WooShippingCreateLabelsView {
         }
     }
 
+    func paymentMethod(_ line: WooShippingPaymentMethodLine) -> some View {
+        VStack(alignment: .leading, spacing: Layout.verticalSpacing) {
+            Text(Localization.BottomSheet.paymentMethod)
+                .footnoteStyle()
+            switch line {
+            case .add:
+                addPaymentMethodLine
+            case .card(let cardLineViewModel):
+                cardPaymentMethodLine(cardLineViewModel)
+            }
+        }
+    }
+
+    var addPaymentMethodLine: some View {
+        Button(action: {
+            /// Trigger payment method selection
+        }) {
+            HStack {
+                Image(systemName: "plus")
+                    .font(.subheadline)
+                    .fontWeight(.semibold)
+                Text(Localization.BottomSheet.addPaymentMethod)
+                    .font(.body)
+                    .fontWeight(.regular)
+                    .foregroundColor(.primary)
+                Spacer()
+            }
+        }
+        .frame(idealHeight: Layout.rowHeight)
```

Comment:
The use of `idealHeight` would collapse the content when a large font size is used. We should avoid it when possible. I logged a separate issue for the accessibility of the Shipment details sheet in WOOMOB-476 to fix other rows, but please consider removing the usage here.